symplyphysics/laws/quantum_mechanics/schrodinger/time_dependent_equation_in_one_dimension.py

```diff
@@ -0,0 +1,30 @@
+from sympy import Eq, sqrt, Derivative, I
+from sympy.physics.units import hbar
+from symplyphysics import (
+    units,
+    Symbol,
+    Function,
+    symbols,
+    clone_symbol,
+)
+
+# Description
+## The Schrödinger equation is a linear partial differential equation that governs the wave
+## function of a quantum-mechanical system. This law describes the general case of a time-dependent
+## potential and a time-dependent wave function.
+
+# Notes
+## - This law works in the case of 1 spatial dimension. To use it for the 3-dimensional space
+##   replace the spatial second derivative with the Laplace operator.
+
+wave_function = Function("wave_function", 1 / sqrt(units.length))
+potential_energy = Function("potential_energy", units.energy)
+particle_mass = clone_symbol(symbols.basic.mass, "particle_mass")
+position = Symbol("position", units.length)
+time = Symbol("time", units.time)
+
+law = Eq(
+    -1 * hbar**2 / (2 * particle_mass) * Derivative(wave_function(position, time), position, 2)
+    + potential_energy(position, time) * wave_function(position, time),
```

Comment:
It seems you are missing brackets.

Reply:
Added forgotten law formula. It should be fine

Reply:
On Wikipedia there are brackets on first term, ie psi is multiplied to sum.

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/2aae5cb7-f2b6-48b0-8a33-07b2a906914c)
You mean this? The first term of the sum is an operator, it's represented by `Derivative(wave_function(position), position, 2)` in the law. If we were to use operator arithmetic, we would need to use sympy or derive our own classes for that.

Reply:
So I'm not missing anything, I've only expanded this expression.

Reply:
Oh, my bad.